content/configuration/devtool.md

```diff
@@ -26,53 +26,84 @@ Choose a style of [source mapping](http://blog.teamtreehouse.com/introduction-so
 
-devtool                       | build | rebuild | production | quality
------------------------------ | ----- | ------- | ---------- | -----------------------------
-eval                          | +++   | +++     | no         | generated code
-cheap-eval-source-map         | +     | ++      | no         | transformed code (lines only)
-cheap-source-map              | +     | o       | yes        | transformed code (lines only)
-cheap-module-eval-source-map  | o     | ++      | no         | original source (lines only)
-cheap-module-source-map       | o     | -       | yes        | original source (lines only)
-eval-source-map               | --    | +       | no         | original source
-source-map                    | --    | --      | yes        | original source
-inline-source-map             | --    | --      | no         | original source
-hidden-source-map             | --    | --      | yes        | original source
-nosources-source-map          | --    | --      | yes        | without source content
-
-T> `+` means faster, `-` slower and `o` about the same time
-
-Some of these values are suited for development and some for production. For development you typically want fast Source Maps at the cost of bundle size, but for production you want separate Source Maps that are accurate.
+T> Instead of using the `devtool` option you can also use the `SourceMapDevToolPlugin`/`EvalSourceMapDevToolPlugin` directly. It has more options. You must never use both `devtool` option and plugin!
+
+devtool                        | build | rebuild | production | quality
+------------------------------ | ----- | ------- | ---------- | -----------------------------
+(none)                         | +++   | +++     | no         | bundled code
+eval                           | +++   | +++     | no         | generated code
+cheap-eval-source-map          | +     | ++      | no         | transformed code (lines only)
+cheap-module-eval-source-map   | o     | ++      | no         | original source (lines only)
+eval-source-map                | --    | +       | no         | original source
+cheap-source-map               | +     | o       | no         | transformed code (lines only)
+cheap-module-source-map        | o     | -       | no         | original source (lines only)
+inline-cheap-source-map        | +     | o       | no         | transformed code (lines only)
+inline-cheap-module-source-map | o     | -       | no         | original source (lines only)
+source-map                     | --    | --      | yes        | original source
+inline-source-map              | --    | --      | no         | original source
+hidden-source-map              | --    | --      | yes        | original source
+nosources-source-map           | --    | --      | yes        | without source content
+
+T> `+++` super fast, `++` pretty fast, `+` fast, `o` medium, `-` slow, `--` pretty slow
```

Comment:
Interesting, I guess it labels it as a 'weasel word', which is kind of true, but an indication there are times you might want it 😆 .

Thanks!